embassy-stm32/src/timer/mod.rs

```diff
@@ -947,7 +949,14 @@ foreach_interrupt! {
         impl_1ch_cmp_timer!($inst);
         impl_2ch_cmp_timer!($inst);
         impl BasicInstance for crate::peripherals::$inst {}
-        impl CaptureCompare16bitInstance for crate::peripherals::$inst {}
+        impl CaptureCompare16bitInstance for crate::peripherals::$inst {
+            /// Enable timer outputs.
+            fn enable_outputs(&self) {
+                use crate::timer::sealed::AdvancedControlInstance;
+                let r = Self::regs_advanced();
+                r.bdtr().modify(|w| w.set_moe(true));
```

Comment:
You might need to copy this code (or `.set_moe()`) to other 2 blocks: `($inst:ident, timer, TIM_1CH_CMP, UP, $irq:ident)`, `($inst:ident, timer, TIM_2CH_CMP, UP, $irq:ident)`, to make sure they also enable `MOE`.

Reply:
Thanks, I would have missed these locations.
Added `fn enable_outputs(&self)` to these timers, also.